src/phel/json.phel

```diff
@@ -0,0 +1,48 @@
+(ns phel\json
+  (:use \Exception))
```

Comment:
What about using `\RuntimeException` to provide some more meaning, instead of `\Exception`?
In such a case, you need to update the line 14 (and others) to ` (throw (php/new RuntimeException ...`

Reply:
I have no problem with this.